pkg/cli/walletcommands.go

```diff
@@ -64,11 +67,39 @@ func createWalletCMD(args []string, publisher wire.EventBroker, rpcBus *wire.RPC
 	fmt.Fprintf(os.Stdout, "Wallet created successfully!\n")
 	fmt.Fprintf(os.Stdout, "Public Address: %s\n", pubAddr)
 
-	cliWallet = w
-	DBInstance = db
+	c.transactor = transactor.New(w, nil)
+
+	if !cfg.Get().General.WalletOnly {
+		go initiator.LaunchConsensus(c.eventBroker, c.rpcBus, w, c.counter)
```

Comment:
nitpick: If LaunchConsensus only starts another go-routines, do we need `go` here?

Reply:
Good catch, I think I did it cause initially I was planning on syncing to tip before starting the other goroutines, though it is only needed for the provisioner component.